lib/tools/system-calls.js

```diff
@@ -771,6 +772,21 @@ systemCallMethods.waitForDevice = async function (appDeviceReadyTimeout = 30) {
   });
 };
 
+systemCallMethods.privilegedShell = async function (args) {
```

Comment:
function names must include verbs

Reply:
docstring is missing

Reply:
maybe `elevateShellPrivileges` or similar?

Reply:
`runPrivilegedShell` ?

Reply:
Are we good with this naming?